distributed/tests/test_client.py

```diff
@@ -7311,3 +7314,96 @@ async def test_dump_cluster_state_error(c, s, a, b, tmp_path):
     )
     assert isinstance(state["workers"][b.address], dict)
     assert state["versions"]["workers"].keys() == {b.address}
+
+
+class TestClientSecurityLoader:
```

Comment:
I'm curious why we're using a test class here. There may be a good reason, it just looks a bit strange as they aren't commonly used throughout `distributed`

It also looks like these aren't being picked up by `pytest` in CI

Reply:
> It also looks like these aren't being picked up by pytest in CI

? I see them in the CI logs.

> I'm curious why we're using a test class here. There may be a good reason, it just looks a bit strange as they aren't commonly used throughout distributed

I've started using this in other projects to help organize tests. I find if I have a 7000 line test file, it's hard to find the tests related to specific functionality. By grouping tests related to a single feature into a class, it's clearer where additional tests should be put (rather than grepping the file to find a related test function to put the new test next to).

I'm happy to move these to just function tests if needed to match the current conventions, but I personally prefer this pattern now.

Reply:
> ? I see them in the CI logs.

Whoops, you're right. I must have done a bad job searching through the logs.